src/ansible_navigator/configuration_subsystem/configurator.py

```diff
@@ -234,7 +234,17 @@ def _apply_cli_params(self) -> None:
                 self._messages.append(LogMessage(level=logging.INFO, message=message))
 
     def _post_process(self) -> None:
+        delayed = []
+        normal = []
+
+        # Separate delayed and normal entries.
```

Comment:
```suggestion
        # Separate normal and delayed entries so they can be processed in that order.
```

Reply:
This function doesn't have a docstring, which is 100% on me. If it did, I'd suggest in there, so it's perfect here.

Reply:
I added the text because I had to look twice at it to see why the lists were even created if they just got put back together :)